build.gradle.kts

```diff
@@ -28,3 +31,11 @@ tasks {
         }
     }
 }
+
+tasks.dokkaHtmlMultiModule.configure {
+    val currentYear = LocalDate.now(ZoneId.of("UTC")).year
+    pluginsMapConfiguration.put(
+        "org.jetbrains.dokka.base.DokkaBase",
+        """ { "footerMessage": "Copyright ${if (2022 == currentYear) currentYear else "2022-$currentYear"} Exactpro Systems Limited" } """,
```

Comment:
Remove checks for 2022. These checks will work only today and tomorrow. It doesn't make sense to add this code